### ISSUE TITLE
Permission Issue (Deleting Servers)

### ISSUE THREAD

**Issue by InflamedSebi** [PufferPanel/PufferPanel#990]
At least one of my users (didn't check others yet) cannot delete servers besides he having all necessary permissions.
In user settings there is "Delete Servers":
![image](https://user-images.githubusercontent.com/1872197/107363077-f07b6c00-6ad9-11eb-9d21-e9765a2e9ade.png)
in server settings everything is checked:
![image](https://user-images.githubusercontent.com/1872197/107363209-1d2f8380-6ada-11eb-9745-2d5b2a81ce2d.png)
But no delete option at the bottom of the server page:
![image](https://user-images.githubusercontent.com/1872197/107363334-47814100-6ada-11eb-8c6e-763470a1012c.png)

**Comment by InflamedSebi** [comment]
Imma just close this, since I deleted the server from DB and filesystem manually and it never happened again and apparently noone else has had this. So I guess it was a local problem.

**Comment by LexFuturorum** [comment]
The delete option is located in the admin tab and therefore not accessable for the user.

**Comment by InflamedSebi** [comment]
> The delete option is located in the admin tab and therefore not accessible for the user.

You are absolutely correct! If I remember correctly this was before the UI Update, but with the new UI makes the "Delete Server" permission pointless:
If you have it you still cannot delete the server because you cannot access the admin panel in the server.
So you additionally need to have admin permission, to access the delete button, in which case you would't need the "Delete Server" permission, since you are already admin.

So this is something to be checked.
The restarting toggles and the delete-button should be accessible by users with appropriate permissions, while editing server definition is definitely an admin feature.
